File: YAzyki programmirovaniya/5 zadach' po fizike na 7 yazykakh programmirovaniya/Задачи VB/WindowsApplication1/Form6.vb
Public Class Form6
    Dim F, a, m, v0, vк, t As Double
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        v0 = Val(textBox2.Text)
        vк = Val(textBox3.Text)
        t = Val(textBox4.Text)
        m = Val(textBox6.Text)

        a = (vк - v0) / t
        F = m * a

        textBox1.Text = CStr(F)
        textBox5.Text = CStr(a)
    End Sub

    Private Sub Form6_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class

File: programming languages/5_physics_problems_in_7_programming_languages/Задачи C#/Задачи/Form6.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Задачи
{
    public partial class Form6 : Form
    {
        public Form6()
        {
            InitializeComponent();
        }
        double F, a, m, v0, vк, t;

        private void Form6_Load(object sender, EventArgs e)
        {

        }

        private void button1_Click(object sender, EventArgs e)
        {
           
            v0 = Convert.ToDouble(textBox2.Text);
            vк = Convert.ToDouble(textBox3.Text);
            t = Convert.ToDouble(textBox4.Text);
            m = Convert.ToDouble(textBox6.Text);
            a = (vк - v0) / t;
            F = m * a;


            textBox1.Text = Convert.ToString(F);
            textBox5.Text = Convert.ToString(a);
        }
    }
}
